docker/README.md

```diff
@@ -0,0 +1,23 @@
+## Running the web tool with maurolepore/r2diirunner-with-packages
+
+[Video](https://asciinema.org/a/364647)
+
+* Ensure your working directory is PACTA_analysis
+
+* Update local repos to the latest commit on the main branch:
+
+```bash
+./bin/update-repos
+```
+
+* Run the web tool in a single step
+
+```bash
+./bin/run-web-tool
```

Comment:
is this an existing file, because I don't see it in this PR?

Reply:
See https://github.com/2DegreesInvesting/PACTA_analysis/blob/docker/docker/README.md